api/src/functionalTest/java/uk/gov/hmcts/payment/functional/PaymentStatusFunctionalTest.java

```diff
@@ -257,6 +259,312 @@ public void return_404_Get_for_payment_failure() {
+        PaymentStatusChargebackDto paymentStatusChargebackDto
+                = PaymentFixture.chargebackRequest(paymentDto.getReference());
+
+        Response chargebackResponse = paymentTestService.postChargeback(
+                SERVICE_TOKEN_PAYMENT,
+                paymentStatusChargebackDto);
+
+        PaymentFailureResponse paymentsFailureResponse =
+                paymentTestService.getFailurePayment(USER_TOKEN_PAYMENT, SERVICE_TOKEN, paymentStatusChargebackDto.getPaymentReference()).then()
+                        .statusCode(OK.value()).extract().as(PaymentFailureResponse.class);
+
+        assertThat(paymentsFailureResponse.getPaymentFailureList().get(0).getPaymentFailureInitiated().getFailureReference()).isEqualTo(paymentStatusChargebackDto.getFailureReference());
+
+        assertThat(refundResponse.getStatusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(chargebackResponse.getStatusCode()).isEqualTo(HttpStatus.OK.value());
+
+        // Ping 2
+        PaymentStatusUpdateSecond paymentStatusUpdateSecond = PaymentStatusUpdateSecond.paymentStatusUpdateSecondWith()
+                .representmentStatus(RepresentmentStatus.No)
+                .representmentDate("2022-10-10T10:10:10")
+                .build();
+        Response ping2Response = paymentTestService.paymentStatusSecond(
+                SERVICE_TOKEN_PAYMENT, paymentStatusChargebackDto.getFailureReference(),
+                paymentStatusUpdateSecond);
+
+        assertEquals(ping2Response.getStatusCode(), OK.value());
+        assertEquals("Successful operation", ping2Response.getBody().prettyPrint());
+
+        // delete payment record
+        paymentTestService.deletePayment(USER_TOKEN, SERVICE_TOKEN, paymentDto.getReference()).then().statusCode(NO_CONTENT.value());
+
+        //delete Payment Failure record
+        paymentTestService.deleteFailedPayment(USER_TOKEN, SERVICE_TOKEN, paymentStatusChargebackDto.getFailureReference()).then().statusCode(NO_CONTENT.value());
+    }
+
+    @Test
+    public void positive_paymentStatusSecond_bulk_scan() {
+
+        // Create a Bulk scan payment
+        String ccdCaseNumber = "1111221233124419";
+        FeeDto feeDto = FeeDto.feeDtoWith()
+                .calculatedAmount(new BigDecimal("550.00"))
+                .ccdCaseNumber(ccdCaseNumber)
+                .version("4")
+                .code("FEE0002")
+                .description("Application for a third party debt order")
+                .build();
+
+        TelephonyCardPaymentsRequest telephonyPaymentRequest =
+                TelephonyCardPaymentsRequest.telephonyCardPaymentsRequestWith()
+                        .amount(new BigDecimal("550"))
+                        .ccdCaseNumber(ccdCaseNumber)
+                        .currency(CurrencyCode.GBP)
+                        .caseType("DIVORCE")
+                        .returnURL("https://www.moneyclaims.service.gov.uk")
+                        .build();
+
```

Comment:
@sayali-salunkhe can you please change the payment method to cheque instead Telephony

Reply:
Updated